src/xdocs/cmdline.xml.vm

```diff
@@ -162,24 +162,24 @@ java -cp checkstyle-${projectVersion}-all.jar com.puppycrawl.tools.checkstyle.gu
     <section name="Run after compilation">
       <p>
         Download and compile:
-        <source>
+        <source class="wrap-content">
 git clone https://github.com/checkstyle/checkstyle.git
 cd checkstyle
 mvn clean compile
         </source>
 
         Run validation with arguments:
-        <source>
+        <source class="wrap-content">
```

Comment:
please remove from source " \"s and combine in one line, ones we have wrapping we could keep it as one line.
Please do the same for all other cases below.